app/src/main/java/com/hedvig/app/BaseActivity.kt

```diff
@@ -22,7 +23,9 @@ abstract class BaseActivity : AppCompatActivity {
     private val marketManager: MarketManager by inject()
 
     override fun onResume() {
-        tracker.screenView(javaClass.simpleName)
+        val currentClassSimpleName = javaClass.simpleName
+        tracker.screenView(currentClassSimpleName)
+        d { "Navigated to $currentClassSimpleName" }
```

Comment:
```suggestion
        tracker.screenView(javaClass.simpleName)
```

If we want to log analytics events, we should do so by adding a `TrackerSink` in debug-builds 🙂

Reply:
I don't think I follow you here.
The one thing I added was `d { "Navigated to $currentClassSimpleName" }`, I am logging the activity change because it makes it super easy for me to open the log and see which screen I am inside to navigate to the appropriate class. Slimber makes it so that this should be free for production builds right? What is the issue with this one?

Reply:
It will indeed be free, but I think a more sustainable way to do this would be to add a `TrackerSink` that logs all analytics events, that way we do not have to manually add these kinds of log-statements at locations where we track 🙂

Reply:
the `TrackerSink` could look something like this:

```kotlin
class DebugLogTrackerSink: TrackerSink {
	override fun track(eventName: String, properties: JSONObject?) {
        d { "Track $eventName, properties: ${properties}" }
    }
    override fun identify(id: String) {
        d { "Identify: $id" }
    }
}
```